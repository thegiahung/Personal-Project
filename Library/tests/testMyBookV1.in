add collection mybooks.csv
add member Python
add member Java
add member C
rent 100000 111112
rent 100000 111113
rent 100001 111114
rent 100002 111115
Member books 100000
member books 100001
relinquish all 100000
relinquish all 100001
member history 100000
member history 100001
exit
