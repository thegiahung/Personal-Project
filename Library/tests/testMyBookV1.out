user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Accel World (Reki Kawahara)
Your Lie In April (Fuji TV)

user: Date A Live (Koshi Tachibana)

user: Success.

user: Success.

user: Accel World (Reki Kawahara)
Your Lie In April (Fuji TV)

user: Date A Live (Koshi Tachibana)

user: Ending Library process.
